test/integration/features/cmd-image.feature

```diff
@@ -0,0 +1,84 @@
+@cmd-image @command
+Feature: Basic image caching test
+  As a user I am able to import and export container images from a local OCI repository
+  located in the $MINISHIFT_HOME/cache directory
+
+  Scenario: As a user I can export a container image from a running Minishift instance
+    Given Minishift has state "Does Not Exist"
+      And executing "minishift delete --force --clear-cache" succeeds
+     When executing "minishift image list" succeeds
+     Then stdout should be empty
+
+     When executing "minishift image export alpine:latest" succeeds
+     Then stdout should contain
+     """
+     Running this command requires an existing 'minishift' VM, but no VM is defined.
+     """
+
+     # Exporting the image
```

Comment:
Blocks like this and below should be scenarios I think. I can make a PR to your branch, or we can fix it soon after the merge.

Reply:
Here we have to disagree. The scenario is "exporting image". This is the actual meat of the scenario. If you don't like the additional comment I rather remove that.